quickie/src/main/kotlin/io/github/g00fy2/quickie/QROverlayView.kt

```diff
@@ -85,25 +85,36 @@ internal class QROverlayView @JvmOverloads constructor(
     super.onDraw(canvas)
   }
 
-  fun setCustomTextAndIcon(stringRes: Int, drawableRes: Int) {
+  fun setCustomText(stringRes: Int) {
     if (stringRes != 0) {
       try {
         binding.titleTextView.setText(stringRes)
       } catch (ignore: NotFoundException) {
         // string resource not found
       }
     }
+  }
+
+  fun setCustomIcon(drawableRes: Int?) {
     if (drawableRes != 0) {
```

Comment:
Currently detekt reports the `NestedBlockDepth` in this function. Thats why the CI does not pass. I would suggest to change this logtic to something like:

if (drawableRes == null) {
binding.titleTextView.setCompoundDrawables(null, null, null, null)
} else if (drawableRes != 0) {
...